.env

```diff
@@ -0,0 +1,14 @@
+BCROS_BASE_URL = "http://localhost:8080/"
```

Comment:
Don't add this file

Reply:
Why not? The app doesn't run correctly locally if I don't have this file.

Reply:
OK, as discussed in RC, I will not add this file (but I will call this `.env.example` instead).

Reply:
Deleted.